src/user/MOM_wave_interface.F90

```diff
@@ -964,8 +964,8 @@ subroutine Update_Stokes_Drift(G, GV, US, CS, dz, ustar, dt, dynamics_step)
           Bottom = 0.0
           do k = 1,GV%ke
             Top = Bottom
-            MidPoint = Bottom - 0.25*(dz(i,J,k)+dz(i,J-1,k))
-            Bottom = Bottom - 0.5*(dz(i,J,k)+dz(i,J-1,k))
+            MidPoint = Bottom - 0.25*(dz(i,j,k)+dz(i,j+1,k))
+            Bottom = Bottom - 0.5*(dz(i,j+1,k)+dz(i,j+1,k))
```

Comment:
The expression for `Bottom` here is wrong, and should read `Bottom = Bottom - 0.5*(dz(i,j,k)+dz(i,j+1,k))`.

Reply:
Thanks a lot for pointing this out. I have now corrected this. Please redo the review.